src/main/scala/uk/gov/homeoffice/drt/AppEnvironment.scala

```diff
@@ -44,8 +44,8 @@ object AppEnvironment {
     ).parMapN(PostgreSQLConfig)
 
   val httpClientConfig: ConfigValue[HttpClientConfig] = (
-    env("HTTP_CLIENT_CONNECTION_TIMEOUT_SECONDS").as[Int].default(2),
-    env("HTTP_CLIENT_REQUEST_TIMEOUT_SECONDS").as[Int].default(2)
+    env("HTTP_CLIENT_CONNECTION_TIMEOUT_SECONDS").as[Int].default(60),
+    env("HTTP_CLIENT_REQUEST_TIMEOUT_SECONDS").as[Int].default(60)
```

Comment:
Are queries taking this long to execute? If so then we need to look at how they're structured and if there are indexes that should be added. Queries really should mostly take a few milliseconds

Reply:
yeah i can change that to 2 seconds it is not issue . I was testing few config here